docs/contributors/readme.md

```diff
@@ -0,0 +1,15 @@
+# Contributors Guide
+
+Welcome to the Gutenberg Project Contributors Guide.
+
+The following guidelines are in place to create consistency across the project and the numerous contributors. See also the [Contributing Documentation](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for technical details around setup, and submitting your contributions.
+
+
+* [Coding Guidelines](../contributors/coding-guidelines.md) outline additional patterns and conventions used in the Gutenberg project.
+
+* [Copy Guidelines](../contributors/copy-guide.md)
+
+* [Design Principles & Vision](../contributors/design.md)
+
+
```

Comment:
Extra newline... our parser is generous in how it translates newlines to `<br>` (some markdown parsers will ignore that extra space, but ours seems more literal) so it's good to keep double line breaks out if possible.

Reply:
👍